fastplotlib/graphics/selectors/_linear.py

```diff
@@ -18,6 +19,19 @@
 
 
 class LinearSelector(Graphic, BaseSelector):
+    @property
+    def limits(self) -> Tuple[float, float]:
+        return self._limits
+
+    @limits.setter
+    def limits(self, values: Tuple[float, float]):
+        if len(values) != 2 or not all(map(lambda v: isinstance(v, Real), values)):
```

Comment:
add code comment here for `not all(map(lambda v: isinstance(v, Real), values))`

pretty ambiguous as is, comment will be helpful later to remember what is happening

Reply:
checks that `values` is an iterable of two real numbers